x/ibc/07-tendermint/types/codec.go

```diff
@@ -24,3 +26,12 @@ func RegisterCodec(cdc *codec.Codec) {
 func SetSubModuleCodec(cdc *codec.Codec) {
 	SubModuleCdc = cdc
 }
+
+// RegisterInterfaces registers the tendermint concrete evidence and client-related
+// implementations and interfaces.
+func RegisterInterfaces(registry types.InterfaceRegistry) {
```

Comment:
@aaronc @alexanderbez do you know if I still need to register the Msgs above even if we are not using proto in this submodule? Same applies for `ClientState`, `ConsensusState` and `Header`.